src/docs/product/relay/metrics.mdx

```diff
@@ -1,54 +1,28 @@
 ---
-title: Errors and Metrics
-description: "Enable error reporting to your project in Sentry."
+title: Metrics
+description: "Configure metrics reporting from your Relay instance."
 sidebar_order: 3
 redirect_from:
   - /meta/relay/metrics/
 ---
-By default, Relay logs errors to the configured logger. You can enable error
-reporting to your project in Sentry in the Relay configuration file:
-
-```yaml
-sentry:
-  enabled: true
-  dsn: <your_dsn>
-```
-
 ## Configuration
```

Comment:
This currently renders two headings underneath each other. What if we remove the headings, and just have the text with an example, and then underneath the heading with metrics.

```suggestion
```